src/client/src/app/site/[site]/view-availability/daily-appointments/page.tsx

```diff
@@ -61,6 +62,23 @@ const Page = async ({ params, searchParams }: PageProps) => {
             displayAction={false}
           />
         </Tab>
+        <Tab title="Manual Cancellations">
+          {orhpanedAppointments.length > 0 ? (
+            <p>
+              {orhpanedAppointments.length} booked appointments are affected.
+              You'll need to manually cancel these appointments.
+            </p>
+          ) : (
+            <p>
+              There are no booked appointments affected by availability changes.
+            </p>
+          )}
+          <DailyAppointmentsPage
+            bookings={orhpanedAppointments}
+            site={site.id}
+            displayAction
```

Comment:
We don't yet support cancelling Orphaned appointments so we shouldn't be displaying the `Cancel` buttons just yet - I think displayAction should be false for now. 

(Also whilst we're here, if we're already checking for orphanedAppointments length being 0, we could add this into the ternary so we don't bother rendering it at all if we don't need to)